plugins/out_opentelemetry/opentelemetry.c

```diff
@@ -723,6 +731,37 @@ static int flush_to_otel(struct opentelemetry_context *ctx,
     resource_log.n_scope_logs = 1;
     resource_logs[0] = &resource_log;
 
+    kv_size = mk_list_size(&(ctx->kv_labels));
+    attributes_list = flb_calloc(kv_size, sizeof(Opentelemetry__Proto__Common__V1__KeyValue *));
+    if (attributes_list == NULL) {
+            flb_errno();
+            return -1;
+    }
+    attributes = flb_calloc(kv_size, sizeof(Opentelemetry__Proto__Common__V1__KeyValue));
+    if (attributes == NULL) {
+            flb_errno();
+            return -1;
+    }
+    for(index = 0; index < kv_size; index++) {
+            attributes_list[index] = &attributes[index];
+    }
+
+
+    kv_index = 0;
+    mk_list_foreach(kv_head, &ctx->kv_labels) {
+        kv = mk_list_entry(kv_head, struct flb_kv, _head);
+        opentelemetry__proto__common__v1__key_value__init(&attributes[kv_index]);
+        attributes[kv_index].key = kv->key;
+        attributes[kv_index].value = otlp_any_value_initialize(MSGPACK_OBJECT_STR, 0);
```

Comment:
We need to verify the result of `otlp_any_value_initialize` here and have a proper cleanup sequence.

Reply:
Maybe this loop could be merged with the one above.